auslib/admin/views/releases.py

```diff
@@ -303,6 +303,39 @@ def _delete(self, release, changed_by, transaction):
         return Response(status=200)
 
 
+class ReleaseReadOnlyView(AdminView):
+    """/releases/:release/read_only"""
+
+    def get(self, release):
+        releases = dbo.releases.getReleases(name=release, limit=1)
+        is_release_read_only = dbo.releases.isReadOnly(name=release, limit=1)
+        if not releases:
+            return Response(status=404,
+                            response='Requested release does not exist')
+        release = releases[0]
+        return jsonify({
+            'read_only': is_release_read_only
+        })
+
+    @requirelogin
+    @requirepermission('/releases/:name/read_only')
+    def _put(self, release, changed_by, transaction):
+        req_read_only = request.form.get('read_only')
```

Comment:
It would be good to use a Form for this, like we do for everything else (see auslib/admin/views/forms.py). The Form can handle validation and coercing the passed value to bool for you.